File: src/Lykke.Service.PayInvoicePortal/Models/Invoices/HistoryItemModel.cs
using System;
using System.Collections.Generic;

namespace Lykke.Service.PayInvoicePortal.Models.Invoices
{
    public class HistoryItemModel
    {
        public string Author { get; set; }

        public string Status { get; set; }

        public double PaymentAmount { get; set; }

        public double SettlementAmount { get; set; }

        public double PaidAmount { get; set; }

        public string PaymentAsset { get; set; }

        public string SettlementAsset { get; set; }

        public int PaymentAssetAccuracy { get; set; }

        public int SettlementAssetAccuracy { get; set; }

        public double ExchangeRate { get; set; }

        public List<string> SourceWalletAddresses { get; set; }

        public string RefundWalletAddress { get; set; }

        public decimal RefundAmount { get; set; }

        public DateTime DueDate { get; set; }

        public DateTime? PaidDate { get; set; }

        public DateTime Date { get; set; }
    }
}

File: src/Lykke.Service.PayInvoicePortal/wwwroot/ng2/src/app/components/InvoiceDetails/History/HistoryItemModel.ts
export class HistoryItemModel {
  public author: string;
  public status: string;
  public paymentAmount: number;
  public settlementAmount: number;
  public paidAmount: number;
  public paymentAsset: string;
  public settlementAsset: string;
  public paymentAssetAccuracy: number;
  public settlementAssetAccuracy: number;
  public exchangeRate: number;
  public sourceWalletAddresses: string[];
  public refundWalletAddress: string;
  public refundAmount: number;
  public dueDate: Date;
  public paidDate?: Date;
  public date: Date;
}
